These paths, or at least some of them, are compiled into kpsewhich so we
need to fix them.
--- ./texk/kpathsea/texmf.cnf.orig	2020-03-21 22:04:56.000000000 +0000
+++ ./texk/kpathsea/texmf.cnf	2020-04-30 07:50:35.805542154 +0000
@@ -59,7 +59,7 @@
 % SELFAUTOPARENT (its grandparent = /usr/local/texlive/YYYY), and
 % SELFAUTOGRANDPARENT (its great-grandparent = /usr/local/texlive).
 % Sorry for the off-by-one-generation names.
-TEXMFROOT = $SELFAUTOPARENT
+TEXMFROOT = @TERMUX_PREFIX@/share/texlive
 
 % The main tree of distributed packages and programs:
 TEXMFDIST = $TEXMFROOT/texmf-dist
@@ -382,7 +382,7 @@
 % OSFONTDIR is to provide a convenient hook for allowing TeX to find
 % fonts installed on the system (outside of TeX).  An empty default
 % value would add "//" to the search paths, so we give it a dummy value.
-OSFONTDIR = /please/set/osfontdir/in/the/environment
+OSFONTDIR = @TERMUX_PREFIX@/share/fonts/TTF
 
 % PostScript Type 1 outline fonts.
 T1FONTS = $TEXMFDOTDIR;$TEXMF/fonts/type1//;$OSFONTDIR//
@@ -431,7 +431,7 @@
 PDFTEXCONFIG = $TEXMFDOTDIR;$TEXMF/pdftex/{$progname,}//
 
 % Used by DMP (ditroff-to-mpx), called by makempx -troff.
-TRFONTS = /usr{/local,}/share/groff/{current/font,site-font}/devps
+TRFONTS = @TERMUX_PREFIX@{/local,}/share/groff/{current/font,site-font}/devps
 MPSUPPORT = $TEXMFDOTDIR;$TEXMF/metapost/support
 
 % For xdvi to find mime.types and .mailcap, if they do not exist in
@@ -582,7 +582,8 @@
 $SELFAUTOPARENT/share/texmf/web2c,\
 $SELFAUTOPARENT/texmf-local/web2c,\
 $SELFAUTOPARENT/texmf-dist/web2c,\
-$SELFAUTOPARENT/texmf/web2c\
+$SELFAUTOPARENT/texmf/web2c,\
+@TERMUX_PREFIX@/share/texlive/texmf-dist/web2c\
 }
 %
 % For reference, here is the old brace-using definition:
